AVsitter2/Plugins/AVprop/[AV]menu.lsl

```diff
@@ -627,6 +628,17 @@ default
                 {
                     MTYPE = (integer)part0;
                 }
+				else if (command == "LMSOURCE")
```

Comment:
Please use spaces to indent instead of tabs. Tabs are at 8 columns in most places (including web browsers, i.e. github) and break formatting.